scalding-db/src/main/scala/com/twitter/scalding/db/macros/impl/ColumnDefinitionProviderImpl.scala

```diff
@@ -208,6 +210,7 @@ object ColumnDefinitionProviderImpl {
           case "BIGINT" =>
```

Comment:
GitHub doesn't allow me to leave a comment on  `case "BOOLEAN" | "TINYINT"` above...

Should we also change that to:

```
case "BOOLEAN" => q"""List("BOOLEAN", "BOOL").contains($typeNameTerm)"""
case "TINYINT" => q"""List("BOOLEAN", "BOOL", "TINYINT").contains($typeNameTerm)"""
```

Reply:
I really don't know. I think maybe @ianoc wrote this code initially. I don't know if he has time to glance at it.

Reply:
Here's what Hibernate uses for mapping basic types: https://docs.jboss.org/hibernate/stable/core.old/reference/en/html/mapping-types.html

Reply:
I think we can leave this as-is.

Reply:
I tried looking through this yesterday, but at this point its been so long, i've mostly forgotten at all. I'm mostly shocked its still in use tbh! :)